blivet/devices/partition.py

```diff
@@ -50,6 +50,8 @@
 # in case the default partition size doesn't fit
 FALLBACK_DEFAULT_PART_SIZE = Size("256MiB")
 
+EXTENDED_METADATA_SIZE = Size("1 MiB")
+
```

Comment:
I think we always use a full alignment grain for the extended metadata. If I'm right, hard-coding this to be `1 MiB` is going to cause problems at some point.